File: Baekjoon/solved_ac/class2/2775.py
# 2775: 부녀회장이 될테야

T = int(input())

for _ in range(T):
  k = int(input())
  n = int(input())

  apartment = [[0 for i in range(n)] for j in range(k+1)]

  for i in range(k):
    sum = 0

    for j in range(n):
      if (i == 0): apartment[i][j] = j+1
      sum += apartment[i][j]
      apartment[i+1][j] = sum
  
  print(apartment[k][n-1])

File: Baekjoon/solved_ac/class2/2775.cpp
// 2775: 부녀회장이 될테야

#include <iostream>
using namespace std;

int main() {
  int T;
  cin >> T;

  while (T--) {
    int k, n;
    cin >> k >> n;

    int apartment[k+1][n];
    for (int i=0; i<k; i++) {
      int sum = 0;

      for (int j=0; j<n; j++) {
        if (i == 0) apartment[i][j] = j+1;

        sum += apartment[i][j];
        apartment[i+1][j] = sum;
      }
    }

    cout << apartment[k][n-1] << '\n';
  }

  return 0;
}